cryptography/src/claim_proofs/mod.rs

```diff
@@ -27,3 +27,6 @@ pub fn random_claim<R: RngCore + CryptoRng>(rng: &mut R) -> (CddClaimData, Scope
         },
     )
 }
+
+pub mod mocked;
+pub mod uuid;
```

Comment:
These shouldn't be added to the library, but looks like you've made your decision.

Reply:
You never see a signing library have _public_ API for mocking messages or private keys.

Reply:
Maybe add a comment here and say these 2 modules must not be used in production code.

Reply:
We should merge until https://github.com/PolymathNetwork/cryptography/pull/97 is merged and the testnet and mainnet branches are created. Then this change can go to the testnet and not the mainnet branch

Reply:
That's a good idea.

Reply:
Until `testnet` and `mainnet` branches are not ready, we could just use the dependencies in Core to point to this branch.
In that way, we don't block the testnet in PolymeshCore.
WDYT?

Reply:
Sounds good to me.